lib/utilities/pristine.js

```diff
@@ -95,7 +95,7 @@ function installPristineApp(appName, options) {
     hasBowerComponents,
     skipNpm,
     emberVersion: options.emberVersion || 'canary',
-    emberDataVersion: options.emberDataVersion || 'emberjs/data#master'
+    emberDataVersion: options.emberDataVersion || 'emberjs/data#v3.8.0'
```

Comment:
Can we use:

```suggestion
    emberDataVersion: options.emberDataVersion || '^3.8.0'
```

I'd rather allow drift (using the `latest` tag)...